### DISCUSSION TITLE
mobile, desktop, embedded(iot) and web

### DISCUSSION THREAD

**Discussion by hintsofttech** [dotnet/Comet#207] [Q&A]
will mvu support mobile, desktop, embedded(iot) and web from the same codebase ? 
because everything concludes at cost effectiveness through max-code-reusability across-devices 
from single codebase like flutter and uno-platform is doing at the moment.

**Comment by saint4eva** [comment]
> will mvu support mobile, desktop, embedded(iot) and web from the same codebase ? because everything concludes at cost effectiveness through max-code-reusability across-devices from single codebase like flutter and uno-platform is doing at the moment.

Comet is built on top .NET Maui. So anything that Maui supports, automatically Comet will support - with extra benefit of providing you with an excellent way of developing apps using C# and C# alone.
